curriculum/challenges/english/25-front-end-development/lab-book-organizer/67172b43f84bcd2dec238a3d.md

```diff
@@ -25,7 +25,7 @@ dashedName: build-a-book-organizer
 
 7. You should filter out books written after a certain year such as 1950 from the `books` array and save the filtered array in a new array named `filteredBooks`.
 
-8. You should sort the books in the `filteredBooks` array according to their `releaseYear` in ascending order and save the sorted array in a new array named `filteredBooksSorted`.
+8. You should sort the books in the `filteredBooks` array according to their `releaseYear` in ascending order. Note: the sort method will sort the array in place. Your original array will be changed. 
```

Comment:
The `sort` method was talked about in the previous lecture video. So the language here should reflect that. Also `sort` needs to be in backticks 

```suggestion
8. You should sort the books in the `filteredBooks` array according to their `releaseYear` in ascending order. You learned in a prior lecture video that the `sort()` method will sort the array in place. This means the `filteredBooks` array will be mutated. 
```